robottelo/ui/locators/common.py

```diff
@@ -144,7 +144,7 @@
     "kt_clear_search": (
         By.XPATH, "//button[contains(@ng-click, 'searchCompleted = false')]"),
     "kt_search_no_results": (
-        By.XPATH, "//span[@data-block='no-search-results-message']"),
+        By.XPATH, "//table//span[@data-block='no-search-results-message']"),
```

Comment:
There 2 elements with such `@data-block` but only one is visible, the one in the results table.